### PR TITLE
DSE-492 #comment Add scalafix

### PR DESCRIPTION
**Pull request by dataswifty**

### CODE REVIEW

File: src/main/scala/org/hatdex/hat/api/json/DataStatsFormat.scala [hunk 1]
```diff
@@ -43,8 +43,8 @@ trait DataStatsFormat extends DataDebitFormats {
   implicit protected val dataStorageStatsFormat: Format[DataStorageStats]     = Json.format[DataStorageStats]
   implicit protected val inboundDataStatsFormat: Format[InboundDataStats]     = Json.format[InboundDataStats]
   implicit protected val outboundDataStatsFormat: Format[OutboundDataStats]   = Json.format[OutboundDataStats]
-  implicit protected val richDataDebitFormat                                  = RichDataJsonFormats.richDataDebitFormat
-  implicit protected val dataDebitFormat                                      = RichDataJsonFormats.dataDebitFormat
+  implicit protected val richDataDebitFormat: Format[RichDataDebit]                                  = RichDataJsonFormats.richDataDebitFormat
```
Comment (augustinas) [inline src/main/scala/org/hatdex/hat/api/json/DataStatsFormat.scala:46]: Is it just me who thinks that this looks wrong? The "=" character is floating in the middle of the line

File: src/main/scala/org/hatdex/hat/api/json/HatJsonFormats.scala [hunk 2]
```diff
@@ -88,12 +88,12 @@ trait HatJsonFormats extends HatJsonUtilities with UuidMarshalling with LocalDat
     unlift(ApiBundleDataSourceField.unapply)
   )
 
-  implicit val ApiBundleDataSourceDatasetFormat   = Json.format[ApiBundleDataSourceDataset]
-  implicit val ApiBundleDataSourceStructureFormat = Json.format[ApiBundleDataSourceStructure]
-  implicit val ApiBundleContextlessFormat         = Json.format[ApiBundleContextless]
+  implicit val ApiBundleDataSourceDatasetFormat: OFormat[ApiBundleDataSourceDataset]   = Json.format[ApiBundleDataSourceDataset]
+  implicit val ApiBundleDataSourceStructureFormat: OFormat[ApiBundleDataSourceStructure] = Json.format[ApiBundleDataSourceStructure]
+  implicit val ApiBundleContextlessFormat: OFormat[ApiBundleContextless]         = Json.format[ApiBundleContextless]
```
Comment (augustinas) [inline src/main/scala/org/hatdex/hat/api/json/HatJsonFormats.scala:93]: And here

**Review by augustinas — COMMENTED** [review]
@dataswifty Some of the formatting looks really weird. Any idea what's causing it?

File: src/main/scala/org/hatdex/hat/api/json/HatJsonFormats.scala (hunk 2 above)
Comment (dataswifty) [reply src/main/scala/org/hatdex/hat/api/json/HatJsonFormats.scala:93]: Pretty sure that scalafix added the return type.  That impacts the formatting.

We don't have a Scalafmt rule configured to align on `=` but we can.

File: src/main/scala/org/hatdex/hat/api/json/DataStatsFormat.scala (hunk 1 above)
Comment (dataswifty) [reply src/main/scala/org/hatdex/hat/api/json/DataStatsFormat.scala:46]: See below.

**Review by augustinas — APPROVED** [review]

**Pull request merged**